### PR TITLE
Feat: version field added

### LINKED ISSUE #835: [FEAT] - Show Version Number In `updates`
### What feature would you like to see?

The `/becca update` command should show the current version number. This can be added to the `updateEmbed` within the `handleUpdates` file. A field should be added between the "Latest Updates" and "Next Scheduled Update" fields. The new field should have the name "Current Version" and the value `process.env.npm_package_version` to keep the result dynamically updated. 

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by princeprag**
# Pull Request

## Description

A new field is added so that whenever we run` \becca updates` we see the current version.

## Related Issue

Closes #835 

## Documentation

For _any_ version updates, please verify if the [documentation page](https://www.beccalyria.com/discord-documentation) needs an update. If it does, please [create an issue there](https://github.com/BeccaLyria/discord-documentation/issues/new?assignees=nhcarrigan&labels=%F0%9F%9A%A6+status%3A+awaiting+triage&template=update.md&title=%5BUPDATE%5D) to ensure it is not forgotten.

- [x] My contribution does NOT require a documentation update.
- [ ] My contribution DOES require a documentation update.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,6 +3,6 @@
 config.json
 /prod/
 /.nyc_output/
-
+.idea
```
Comment (nhcarrigan) [inline .gitignore:6]: Go ahead and remove this - this is an editor-specific workspace settings folder.

Comment (princeprag) [reply .gitignore:6]: Okay.. sure! So, we should essentially take care not to push those .vscode .idea files manually.. i usually do `git add *`

**Review by nhcarrigan — APPROVED** [review]
This looks good to me!

**Comment by nhcarrigan** [comment]
## Congratulations

Congratulations on making your first pull request to our project. We have reviewed your contributions and are happy to accept them.

We hope you enjoyed the contribution process, and encourage you to watch for issues labelled `help wanted` for additional opportunities to contribute.

**Comment by princeprag** [comment]
nhverification

**Pull request merged**